bundle/src/main/resources/i18n/de.json

```diff
@@ -45,7 +45,8 @@
         "collection": "Listen-Vererbung aktivieren",
         "collectionItem": "Diese Konfiguration wird über Listen-Vererbung vererbt.",
         "config": "Diese Konfiguration wird vererbt.",
-        "property": "Eigenschafts-Vererbung aktivieren"
+        "property": "Eigenschafts-Vererbung aktivieren",
+        "propertyDescription": "Wenn aktiviert werden Eigenschaften die hier nicht explizit gesetzt sind von Konfigurations aus höheren Vererbungs-Ebenen übernommen."
```

Comment:
```suggestion
        "propertyDescription": "Wenn aktiviert, werden Eigenschaften, die hier nicht explizit gesetzt sind, von Konfigurationen aus höheren Ebenen des Vererbungsbaums übernommen."
```

Reply:
thanks, i've used the better translation